test/util/ReduceCssCalcPOC.spec.ts

```diff
@@ -0,0 +1,135 @@
+import reduceCSSCalc from 'reduce-css-calc';
```

Comment:
this all feels like we're introducing so much complexity for us to manage for very little gain - package size will probably remain similar with these additions